Azure-Samples/digital-twins-samples-csharp
# Readme and Documentation

## Purpose

Improve `README's` and documentation.

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[ ] New sample or new feature within a sample
[ ] Code style update (formatting, naming)
[ ] Refactoring (no functional changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
Get the code
```
git clone https://github.com/Azure-Samples/digital-twins-samples-csharp.git
cd digital-twins-samples-csharp
```

Run the tests
```
dotnet restore <folder>
dotnet build <folder>
dotnet restore <folder>.tests
dotnet test <folder>.tests
```

Also verify anything you think is relevant by manually running the sample(s)
```
cd <folder>
dotnet run
```

## Other Information
